manifests/validator.pp

```diff
@@ -0,0 +1,25 @@
+# == Class: grafana::validator
+#
+# Includes `` resource
+#
+# === Parameters
+# [*grafana_url*]
+# Grafana URL.
+#
+# [*grafana_api_path*]
+# API path to validate with.
+#
+# === Examples
+#
+#  include grafana::validator
+#
+class grafana::validator (
+  String[1] $grafana_url = 'http://localhost:3000',
+  String[1] $grafana_api_path = '/api/health',
```

Comment:
could we use Stdlib::Absolutepath here?

Reply:
It's not a filesystem path through, I think `Stdlib::Absolutepath` is intended for filesystem paths, not URL paths, though I supposed Linux paths look like URL paths.

Reply:
Switched to `Stdlib::Absolutepath`